content/src/ContentMetrics.ts

```diff
@@ -25,3 +25,39 @@ export const REPOSITORY_QUEUE_EXECUTED_QUERIES = new Histogram({
   help: 'Time spent in seconds since the queries were added to the queue until they got resolved',
   labelNames: ['priority']
 })
+
+export const DCL_CONTENT_GARBAGE_COLLECTION_ITEMS_TOTAL = new Counter({
+  name: 'dcl_content_garbage_collection_items_total',
+  help: 'Total number of garbage collected items',
+  labelNames: []
+})
+
+export const DCL_CONTENT_GARBAGE_COLLECTION_TIME = new Histogram({
+  name: 'dcl_content_garbage_collection_time',
+  help: 'Histogram of time spent in garbage collection',
+  labelNames: []
+})
+
+export const DCL_CONTENT_DOWNLOAD_TIME = new Histogram({
```

Comment:
I'd use a name that reflects that this content download is from the sync, something like `DCL_SYNC_CONTENT_DOWNLOAD_TIME`?